scipy/optimize/tests/test__linprog_clean_inputs.py

```diff
@@ -177,7 +181,7 @@ def test__clean_inputs1():
     assert_allclose(lp_cleaned.b_ub, np.array(lp.b_ub))
     assert_allclose(lp_cleaned.A_eq, np.array(lp.A_eq))
     assert_allclose(lp_cleaned.b_eq, np.array(lp.b_eq))
-    assert_(lp_cleaned.bounds == [(0, None)] * 2, "")
+    assert_(np.all(lp_cleaned.bounds == [(0, np.inf)] * 2), "")
```

Comment:
It occurs to me that we should be using `np.testing.assert_equal` or `np.testing.assert_array_equal` here. If this is the only issue, I'll go ahead and fix it then merge.